.gitmodules

```diff
@@ -217,6 +217,10 @@
 [submodule "projects/braincode_POND_Registry_Imaging_Data_Release"]
 	path = projects/braincode_POND_Registry_Imaging_Data_Release
 	url = https://github.com/conpdatasets/braincode_POND_Registry_Imaging_Data_Release
+[submodule "projects/NiMARE__Neuroimaging_Meta_Analysis_Research_Environment"]
+	path = projects/NiMARE__Neuroimaging_Meta_Analysis_Research_Environment
+	url = https://github.com/conp-bot/conp-dataset-NiMARE-Neuroimaging-Meta-Analysis-Research-Environment.git
```

Comment:
This dataset seems associated with [this NeurLibre book](https://neurolibre.herokuapp.com/papers/10.55458/neurolibre.00007).  There is no description in the README or the Zenodo archive the CONP was tagged from.  The same applies to the DATS.json, which I would guess is also failing validation given how little information there is to crawl from the Zenodo archive.

I will communicate with the NeuroLibre team to see if they can at least add the paragraph of description at the beginning of their NeuroLibre book.